outputs.tf

```
@@ -13,3 +13,18 @@ output "realtime_log_config_arn" {
   description = "ARN of the realtime logging configuration"
 
 }
+
+output "certificate_arn" {
+  description = "The ARN of the issued ACM certificate."
+  value       = aws_acm_certificate.cert.arn
+}
+

```

```suggestion
output "certificate_domain_validation_options" {
  description = "The validation options of the issued ACM certificate."
  value           = aws_acm_certificate.cert.cert.domain_validation_options
}

```

In fact from https://github.com/tx-pts-dai/roland-terraform/blob/main/fum-3488/infra/cdn_main.tf#L36C66-L36C91 I need the "domain_validation_options" of the cert